src/identity/components/OrganizationListTab/OrganizationCard.tsx

```diff
@@ -14,17 +15,39 @@ interface OrgCardProps {
   provider: string
   regionCode: string
   regionName: string
+  provisioningStatus: string
 }
 
 export const OrganizationCard: FC<OrgCardProps> = ({
   name,
   provider,
   regionCode,
   regionName,
+  provisioningStatus,
 }) => {
+  const isOrgSuspended = provisioningStatus === 'suspended'
+
+  const tooltipContent = (
+    <p>
+      Organizations can be reactivated within 7 days. Contact support at{' '}
+      <a
+        href="mailto:support@influxdata.com"
+        target="_blank"
+        rel="noreferrer noopener"
+      >
```

Comment:
I'm pretty sure you can use [`SafeBlankLink`](https://github.com/influxdata/ui/blob/master/src/utils/SafeBlankLink.tsx) here since you're passing an `href`

Reply:
yes i can! i'll change it to SafeBlankLink